diff --git a/fs/internal.h b/fs/internal.h
index b4338451c..6c3d8a3b9 100644
--- a/fs/internal.h
+++ b/fs/internal.h
@@ -68,6 +68,7 @@ extern int finish_automount(struct vfsmount *, struct path *);
 extern int sb_prepare_remount_readonly(struct super_block *);
 
 extern void __init mnt_init(void);
+int path_umount(struct path *path, int flags);
 
 extern int __mnt_want_write(struct vfsmount *);
 extern int __mnt_want_write_file(struct file *);
